bindings/go/mio/mkv.go

```diff
@@ -156,7 +156,7 @@ func (mkv *Mkv) doIdxOp(opcode uint32, key []byte, value []byte,
 
     var rcI C.int32_t
     var op *C.struct_m0_op
-    rc := C.m0_idx_op(mkv.idx, opcode, &k, vPtr, &rcI, flags, &op)
+    rc := C.m0_idx_op(mkv.idx, opcode, &k, vPtr, &rcI, 1, flags, &op)
```

Comment:
Do you think we can introduce something like:
ioctl(2) or setsockopt(3).

For example,
m0_idx_op_setoption(struct m0_op *op, char *opname, int opval);

And we can handle the "min_success" option in this way.
In the future, we can handle other options in this way, too.

Reply:
It's a good idea, but I didn't get how it helps in this case, as operation is executed as part of m0_idx_op() where we need this value and "op" too will be created in it.

Reply:
The `op` is created and initialized in the m0_idx_op(), but now it is not submitted.
So, we can initialize the min_success to a default value, here is 1.
If the user of Motr wants to modify this value, he/she must call another API, let's say
m0_idx_op_setoption(struct m0_op *op, char *opname, int opval);
Otherwise, no such call is needed.
Then, the user calls m0_op_launch(&op, 1) && m0_op_wait().


So, no changes are needed to existing code.
If a user wants to have a different value for min_success, he/she needs to call m0_idx_op_setoption() immediately after m0_idx_op().

We need to main compatibility of code. So existing Motr application does not change.

Reply:
looks fine to me, as discussed let's have enum for "char *opname "

Reply:
Thank you.